doc/source/whatsnew/v1.2.1.rst

```diff
@@ -12,6 +12,10 @@ including other versions of pandas.
 
 .. _whatsnew_121.regressions:
 
+We have reverted a commit that resulted in several plotting related regressions in pandas 1.2 (:issue:`38969`, :issue:`38736`, :issue:`38865`, :issue:`38947` and :issue:`39126`)
+
+As a result, bugs reported as fixed in pandas 1.2 are again present (:issue:`26186` and :issue:`11465`)
```

Comment:
I would move this below the list below? (now it's not under the "Fixed regressions" title)

Reply:
we didn't actually 'fix', but will defer to you on documentation layout.

Reply:
The regressions are still fixed, no? (although tests are not yet added, I am assuming this fixed the regression reports)

Reply:
I see your point. I sort of meant we didn't `fix` as we broke other things. but maybe regression section is more appropriate than the introductory sentences.